### PR TITLE
[SWA-99][FEAT] - 0x Protocol API and minor social media updates

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-99](https://linear.app/swaprhq/issue/SWA-99/0x-protocol-not-showing-as-router)

# Description

* Adds 0x Protocol env variable for development purposes
* Updates Twitter links and labels into their new X brand
* Toggles the Stackly toast from old to new (yellow to green) styling

# Visual evidence
image
image
image

# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet
5) 0x Protocol changes are dependant on the new SDK release

### CODE REVIEW

**Comment by Diogomartf** [comment]
can we update the Stackly link to the landing page link? 

https://stackly.app/

Also might be better to put it as a env variable

File: .env.production [hunk 1]
```diff
@@ -1,6 +1,6 @@
 REACT_APP_FATHOM_SITE_ID=KKJYZYXY
 REACT_APP_FATHOM_SITE_SCRIPT_URL=https://gold-stunning.swapr.site/script.js
-REACT_APP_STACKLY_DESIGN_VERSION=old
+REACT_APP_STACKLY_DESIGN_VERSION=new
```
Comment (Diogomartf) [inline .env.production:3]: I think we should remove this variable now and just show the new banner

Comment (ElRodrigote) [reply .env.production:3]: Yes! This change is faster for release purposes, talked about this with @0xVenky and he agreed on me creating a follow-up task to drop all the unused code in another task (post-release):

https://linear.app/swaprhq/issue/SWA-101/clean-all-unused-code-and-env-vars-for-stackly-toast

Comment (Diogomartf) [reply .env.production:3]: ok!

**Review by Diogomartf — APPROVED** [review]

**Pull request merged**